src/header/CatalogHeader.tsx

```diff
@@ -0,0 +1,24 @@
+import Header from '@edx/frontend-component-header';
+
+import { useMenuItems } from './hooks/useMenuItems';
+import { getLogoDestination } from './utils';
+import { AuthenticatedUserTypes } from './types';
+
+const CatalogHeader = () => {
+  const {
+    authenticatedUser,
+    mainMenu,
+    secondaryMenu,
+    isNotHomePage,
```

Comment:
`isNotHomePage` feels confusing to me. After some digging I can see that it is intended to mean "we are not currently on the home page," but passing it in to `getLogoDestination` implies to me "the destination for the logo link is not the home page."

Reply:
Removed at https://github.com/openedx/frontend-app-catalog/pull/24/commits/624f1860b302a7f77ac5352e164a196d2a8976c8